virtcontainers/qemu.go

```diff
@@ -646,17 +646,19 @@ func (q *qemu) stopSandbox() error {
 
 	err := q.qmpSetup()
 	if err != nil {
-		return err
+		q.Logger().WithError(err).Error("Failed to setup qmp")
 	}
 
 	err = q.qmpMonitorCh.qmp.ExecuteQuit(q.qmpMonitorCh.ctx)
 	if err != nil {
 		q.Logger().WithError(err).Error("Fail to execute qmp QUIT")
-		return err
 	}
 
 	// cleanup vm path
 	dir := filepath.Join(store.RunVMStoragePath, q.id)
+	if err := os.RemoveAll(dir); err != nil {
```

Comment:
There's no need to move this, it doesn't change any of the actions that are gonna be performed.